mentat/code_context.py

```diff
@@ -154,29 +129,29 @@ async def get_code_message(
             ]
 
         code_message += ["Code Files:\n"]
-        meta_tokens = count_tokens("\n".join(code_message), model, full_message=True)
 
         # Calculate user included features token size
         include_features = [
             feature
             for file_features in self.include_files.values()
             for feature in file_features
         ]
-        include_files_message = get_code_message_from_features(include_features)
```

Comment:
I moved all this to the if because it's only relevant if auto_context is on. Benchmarking shows its basically unimportant for performance but I think its easier to see how things are used and the flow of logic this way.